Source/Core/Common/CommonPaths.h

```diff
@@ -95,6 +95,7 @@
 #define DUALSHOCKUDPCLIENT_CONFIG "DSUClient.ini"
 #define FREELOOK_CONFIG "FreeLook.ini"
 
+
```

Comment:
Is this an intentional change? The file is otherwise unchanged, and could just be removed from the commit.